src/Castle.Core/DynamicProxy/Generators/Emitters/SimpleAST/Reference.cs

```diff
@@ -45,14 +45,9 @@ public virtual void Generate(ILGenerator gen)
 		{
 		}
 
-		public virtual Expression ToAddressOfExpression()
+		void IExpressionOrStatement.Emit(IMemberEmitter member, ILGenerator gen)
```

Comment:
Why does this need to be explicitly implemented?

Reply:
Because of... uh, important reasons that I can't recall. 😁 Fixed now.